spec/dummy/config/webpack/commonWebpackConfig.js

```diff
@@ -1,11 +1,14 @@
 // Common configuration applying to client and server configuration
-const { generateWebpackConfig, merge } = require('shakapacker');
+const { generateWebpackConfig, webpackConfig: v6WebpackConfig, merge } = require('shakapacker');
+
+const generateWebpackConfigAlias = generateWebpackConfig ? generateWebpackConfig : () => undefined;
+
+const baseClientWebpackConfig = v6WebpackConfig ? v6WebpackConfig : generateWebpackConfigAlias();
```

Comment:
Revert any complexities here and have CI change the file if needed for tests on 6.6 shakapacker